plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -0,0 +1,141 @@
+        });
+
+        describe('Given alice deposited', () => {
+            beforeEach(aliceDeposits);
+
+            describe('When alice starts standard exit on the deposit tx', () => {
+                beforeEach(async () => {
+                    await this.exitGame.startStandardExit(
+                        this.depositUtxoPos, this.depositTx, this.merkleProofForDepositTx,
+                        { from: alice, value: STANDARD_EXIT_BOND },
+                    );
+                });
+
+                it('should saves the StandardExit data when successfully done', async () => {
+                    const exitId = await this.exitIdHelper.getStandardExitId(true, this.depositTx, this.depositUtxoPos);
+                    const standardExitData = await this.exitGame.exits(exitId);
+
+                    expect(standardExitData.exitable).to.be.true;
+                    expect(standardExitData.position).to.be.bignumber.equal(new BN(this.depositUtxoPos));
+                    expect(standardExitData.token).to.equal(ETH);
+                    expect(standardExitData.exitTarget).to.equal(alice);
+                    expect(standardExitData.amount).to.be.bignumber.equal(new BN(DEPOSIT_VALUE));
+                });
+
+                it('should put the exit data into the queue of framework', async () => {
+                    const priorityQueueAddress = await this.framework.exitsQueues(ETH);
+                    const priorityQueue = await PriorityQueue.at(priorityQueueAddress);
+                    const uniquePriority = await priorityQueue.getMin();
+
+                    // right most 64 bits are nonce for priority queue
+                    expect(uniquePriority.shrn(64)).to.be.bignumber.equal(new BN(this.depositUtxoPos));
+                });
+            });
+        });
+
+        describe('Given alice deposited and transfered to bob', () => {
+            beforeEach(async () => {
+                await aliceDeposits();
+                await aliceTransferToBob();
+            });
+
+            describe('When bob tries to start the standard exit on the transfered tx', () => {
+                beforeEach(async () => {
+                    await this.exitGame.startStandardExit(
+                        this.transferUtxoPos, this.transferTx, this.merkleProofForTransferTx,
+                        { from: bob, value: STANDARD_EXIT_BOND },
+                    );
+                });
+
+                it('should start successully', async () => {
+                    const exitId = await this.exitIdHelper.getStandardExitId(
+                        false, this.transferTx, this.transferUtxoPos,
+                    );
+                    const standardExitData = await this.exitGame.exits(exitId);
+                    expect(standardExitData.exitable).to.be.true;
+                });
+            });
+        });
+
+        describe('Given alice deposited and transfered to bob', () => {
```

Comment:
Merge that with previous describe on the same level.

Reply:
This is how it looks currently. 
```
     Given alice deposited and transfered to bob
        When bob tries to start the standard exit on the transfered tx
          ✓ should start successully
      Given alice deposited and transfered to bob
        When alice tries to start the standard exit on the deposit tx
          ✓ should still be able to start standard exit even already spent
          Then bob can challenge the standard exit spent
            ✓ TODO:  should challenges it successfully
```

If we merge, it would be like this
```
     Given alice deposited and transfered to bob
        When bob tries to start the standard exit on the transfered tx
          ✓ should start successully
        When alice tries to start the standard exit on the deposit tx
          ✓ should still be able to start standard exit even already spent
          Then bob can challenge the standard exit spent
            ✓ TODO:  should challenges it successfully
```

`When bob tries to start the standard exit on the transfered tx` and `When alice tries to start the standard exit on the deposit tx` would looks like consecutive steps instead of different test suite.

That's why I duplicate them on purpose.